nnmnkwii/datasets/jvs.py

```diff
@@ -0,0 +1,221 @@
+    Users are expected to inherit the class and implement ``collect_features``
+    method, which defines how features are computed given a transcription.
+
+    Args:
+        data_root (str): Data root.
+        speakers (list): List of speakers to find. Speaker id must be ``str``.
+          For supported names of speaker, please refer to ``available_speakers``
+          defined in the module.
+        labelmap (dict[optional]): Dict of speaker labels. If None,
+          it's assigned as incrementally (i.e., 0, 1, 2) for specified
+          speakers.
+        max_files (int): Total number of files to be collected.
+
+    Attributes:
+        speaker_info (dict): Dict of speaker information dict. Keyes are speaker
+          ids (str) and each value is speaker information consists of ``gender``,
+          ``minf0`` and ``maxf0``.
+        labels (numpy.ndarray): Speaker labels paired with collected files.
+          Stored in ``collect_files``. This is useful to build multi-speaker
+          models.
+
+    """
+    def __init__(self, data_root, speakers=available_speakers, labelmap=None, max_files=None):
+        super(TranscriptionDataSource, self).__init__(data_root, speakers, labelmap, max_files)
+
+    def collect_files(self, nonpara=False, whisper=False):
+        return super(TranscriptionDataSource, self).collect_files(False, nonpara=nonpara, whisper=whisper)
+
+
+class WavFileDataSource(_JVSBaseDataSource):
+    """WavFile data source for JVS dataset.
+
+    The data source collects text transcriptions from JVS.
+    Users are expected to inherit the class and implement ``collect_features``
+    method, which defines how features are computed given a transcription.
+
+    Args:
+        data_root (str): Data root.
+        speakers (list): List of speakers to find. Speaker id must be ``str``.
+          For supported names of speaker, please refer to ``available_speakers``
+          defined in the module.
+        labelmap (dict[optional]): Dict of speaker labels. If None,
+          it's assigned as incrementally (i.e., 0, 1, 2) for specified
+          speakers.
+        max_files (int): Total number of files to be collected.
+
+    Attributes:
+        speaker_info (dict): Dict of speaker information dict. Keyes are speaker
+          ids (str) and each value is speaker information consists of ``gender``,
+          ``minf0`` and ``maxf0``.
+        labels (numpy.ndarray): Speaker labels paired with collected files.
+          Stored in ``collect_files``. This is useful to build multi-speaker
+          models.
+    """
+    def __init__(self, data_root, speakers=available_speakers, labelmap=None, max_files=None):
+        super(WavFileDataSource, self).__init__(data_root, speakers, labelmap, max_files)
+
+    def collect_files(self, nonpara=False, whisper=False):
+        return super(WavFileDataSource, self).collect_files(True, nonpara=nonpara, whisper=whisper)
+
+if __name__ == "__main__":
```

Comment:
Could you remove your local code in the PR? Instead, would be great if you could add tests like for other datasets so that we can run them on CI. See 
- https://github.com/r9y9/nnmnkwii/blob/master/tests/test_datasets.py
- https://github.com/r9y9/nnmnkwii/blob/master/tests/test_real_datasets.py

for reference.